### PR TITLE
email publications@opensafely.org when repos with outputs are signed off

### PR DESCRIPTION
**Pull request by ghickman**
Fixes: #2262

### CODE REVIEW

File: jobserver/emails.py [hunk 1]
```diff
@@ -29,21 +29,35 @@ def send_finished_notification(email, job):
...
+        template_name="emails/notify_researcher_repo_signed_off.txt",
+        context={"repo": repo},
+    )
+
+
+def send_repo_signed_off_notification_to_staff(repo):
+    numbers = [w.project.number for w in repo.workspaces.all() if w.project.number]
+    numbers = ",".join(numbers) if numbers else "X"
+    subject = (
+        f"[{numbers}] - Historic repo with outputs - sign off required {repo.name}"
```
Comment (bloodearnest) [inline jobserver/emails.py:52]:
```suggestion
        f"[{numbers}] - Historic repo with outputs in github - sign off required {repo.name}"
```

File: jobserver/templates/emails/notify_staff_repo_signed_off.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{{ staff_url }}
+
+The repository, {{ repo.name }}, has been signed off by {{ repo.researcher_signed_off_by.name }}.
+
```
Comment (bloodearnest) [inline jobserver/templates/emails/notify_staff_repo_signed_off.txt:4]:
```suggestion

This historic Github repo contains outputs released from a backend, and needs additional sign off.

```

**Review by bloodearnest — APPROVED** [review]
LGTM, couple of minor suggestions to email text

File: jobserver/emails.py (hunk 1 above)
Comment (ghickman) [reply jobserver/emails.py:52]: This is only sent for repos with outputs on GitHub :)

**Pull request merged**